google/cloud/aiplatform/training_jobs.py

```diff
@@ -396,33 +429,97 @@ def _create_input_data_config(
+            if any(
+                [
+                    training_filter_split is not None,
+                    validation_filter_split is not None,
+                    test_filter_split is not None,
+                ]
+            ):
+                if all(
+                    [
+                        training_filter_split is not None,
+                        validation_filter_split is not None,
+                        test_filter_split is not None,
+                    ]
                 ):
+                    filter_split = gca_training_pipeline.FilterSplit(
+                        training_filter=training_filter_split,
+                        validation_filter=validation_filter_split,
+                        test_filter=test_filter_split,
+                    )
+                else:
                     raise ValueError(
-                        "A pre-defined split may only be used with a tabular or time series Dataset"
+                        """All filter splits must be passed together or not at all"""
                     )
 
+            # Create predefined split
+            if predefined_split_column_name:
                 predefined_split = gca_training_pipeline.PredefinedSplit(
                     key=predefined_split_column_name
                 )
 
-            # Create GCS destination
+            # Create timestamp split or fraction split
+            if timestamp_split_column_name:
+                timestamp_split = gca_training_pipeline.TimestampSplit(
+                    training_fraction=training_fraction_split,
+                    validation_fraction=validation_fraction_split,
+                    test_fraction=test_fraction_split,
+                    key=timestamp_split_column_name,
+                )
+            elif any(
+                [
+                    training_fraction_split is not None,
+                    validation_fraction_split is not None,
+                    test_fraction_split is not None,
+                ]
+            ):
+                fraction_split = gca_training_pipeline.FractionSplit(
+                    training_fraction=training_fraction_split,
+                    validation_fraction=validation_fraction_split,
+                    test_fraction=test_fraction_split,
+                )
+
+            splits = [
+                split
+                for split in [
+                    filter_split,
+                    predefined_split,
+                    timestamp_split_column_name,
+                    fraction_split,
+                ]
+                if split is not None
+            ]
+
+            # Fallback to fraction split if nothing else is specified
+            if len(splits) == 0:
+                logging.info(
+                    "No dataset split provided. The service will use a default split."
+                )
+            elif len(splits) > 1:
+                raise ValueError(
+                    """Can only specify one of:
```

Comment:
@sasha-gitg Any thoughts about the wording? I copied from Torry's PR so thought you already reviewed it.

Reply:
Perhaps we can remove the `OR` as it's redundant with the specify one of and the enumerated list.

I think anything more than this would be more complicated.

Reply:
Fixed.